datafusion/substrait/tests/serialize.rs

```diff
@@ -33,15 +33,15 @@ mod tests {
         let sql = "SELECT a, b FROM data";
         // Test reference
         let df_ref = ctx.sql(sql).await?;
-        let plan_ref = df_ref.to_logical_plan()?;
+        let plan_ref = df_ref.into_optimized_plan()?;
```

Comment:
I hadn't noticed when this changed from "to_optimized" to "to_unoptimized". But as discussed in https://github.com/datafusion-contrib/datafusion-substrait/pull/35, maybe we need not to cover optimization in serialize/deserialize phase?